nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,9 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        if colorbar:
+            self._colorbar_show(ax, threshold=edge_threshold)
```

Comment:
> By definition, the classes defined here with suffix "Axes" are supposed to be classes compatible with matplotlib Axes in order to be able to feed them to functions that are supposed to process Axes... So, in my sense, this hack makes sense in that context.

Is that actually true? I thought they were just wrapping a matplotlib Axes through their `.ax` attribute.